cranelift/codegen/src/ctxhash.rs

```diff
@@ -84,6 +84,7 @@ where
 impl<K, V> CtxHashMap<K, V> {
     /// Insert a new key-value pair, returning the old value associated
     /// with this key (if any).
+    #[allow(dead_code)] // Used in testing.
```

Comment:
If this is used only in unit tests within this crate, then this can be:

```suggestion
    #[cfg(test)]
```